documentation/operators/src/legal/landing-pages.md

```diff
@@ -7,3 +7,328 @@ The entire content of this page is under [Creative Commons Attribution 4.0 Inter
 > Nym Node operators are invited to add their legal findings and suggestions directly through [pull requests](add-content.md).
 
 Exit Gateway landing page is a great and transparent way to prevent possible troubles by providing people with basic facts, links to laws and regulations on a given topic as well as operator's contact info. To inspire each other we share some examples how Nym community handled this issue below.
+
+## Avril 14th Exit Gateways
+
+The following snippet it on the 'assets' path under '[http]' section of your config file
```

Comment:
I am not sure if this one has a typo. "The following snippet it on the .." Maybe I just don't understant it well.

Reply:
typooos:)
and quite cryptic too
sorry,
I'm rewording this for the sake of clarity